source/reference/api/reverse.txt

```diff
@@ -152,6 +152,13 @@ To view the mapping direction for the synchronization of the source and
 destination clusters, use the :ref:`progress <c2c-api-progress>`
 endpoint and check the ``directionMapping`` object.
 
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+The embedded verifier is not compatible with reversible sync. If
```

Comment:
Should we indicate that the verification is enabled for the forward direction of the migration before /reverse is called?

Reply:
^^ yes if we can do that, that would be great

Reply:
Fixed